src/components/equations/ItemEffects.test.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from "chai";
+import { stubBlankGame } from "../../fakes.test";
+import { IPokemon } from "../Battles";
+import { ItemEffects } from "./ItemEffects";
+
+describe("ItemEffects", () => {
+    describe("capturePokemon", () => {
+        it("adds a new Pokemon to your party when your party has fewer than six Pokemon", (): void => {
+            // Arrange
+            const { fsp } = stubBlankGame();
+            const pokemon: IPokemon = fsp.equations.newPokemon({
+                level: 5,
+                title: "SQUIRTLE".split(""),
+            });
+            const itemEffects = new ItemEffects(fsp);
+
+            // Act
+            itemEffects.capturePokemon(pokemon);
+            itemEffects.capturePokemon(pokemon);
+
+            // Assert
+            expect(fsp.itemsHolder.getItem(fsp.storage.names.pokemonInParty)[0].title).to.be.equal(pokemon.title);
```

Comment:
A comment was made here earlier. My response was that the party is initialized to be a size of zero, so adding a new Pokemon adds a Pokemon to the first index - so I think it should be valid.

Reply:
:+1: correct! I was wrong.

Reply:
However, you'll need a test for the case of adding a Pokemon to the party when the party already has 1 (not 6) Pokemon in it.

Reply:
I call capturePokemon twice in the tests, so it covers the case when the party has no Pokemon, and when the party already has 1 Pokemon. Am I to assume that you maybe want to split up the test?

Reply:
There are four cases that are relevant herr:

* You have 0 Pokemon
* You have 1 Pokemon
* You have >=2 and <6 Pokemon (so let's say 2)
* You have 6 Pokemon (a full party)